src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -771,6 +772,22 @@
                                                      IsChecked="{Binding Path=NodeAutocompleteMachineLearningIsChecked, Converter={StaticResource BinaryRadioButtonCheckedConverter},ConverterParameter=True}"
                                                      Content="{x:Static p:Resources.RecommendedNodes}"
                                                      IsEnabled="{Binding Path=NodeAutocompleteIsChecked}"/>
+                                                <!--Beta tag-->
+                                                <Label Grid.Column="2"
+                                                    Name="NodeRecommendationBetaTag"
+                                                    Content="{x:Static p:Resources.PreferencesViewBetaTag}"
+                                                    Background="{StaticResource SecondaryGray}"
+                                                    Foreground="{StaticResource PreferencesWindowFontColor}"
+                                                    Margin="10,-5,0,0"
+                                                    FontSize="12"
```

Comment:
The beta tag label looks bigger in size than the actual option. Do you want to size this down?

Reply:
Sure  what's your suggestion, I am less familiar with how to size down lable

Reply:
I was thinking FontSize property was affecting that.

Reply:
Ah OK, I will give it a try, I only used what @Jingyi-Wen specified on Jira

Reply:
I downgraded size to 10
<img width="665" alt="image" src="https://user-images.githubusercontent.com/3942418/205348246-7979cace-a8ea-4c73-9f63-e9f4a2591be4.png">